code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -76,6 +76,8 @@
 	var/list/childtype = null
 	var/next_scan_time = 0
 	var/animal_species //Sorry, no spider+corgi buttbabies.
+	var/current_offspring = 0
+	var/max_offspring = DEFAULT_MAX_OFFSPRING
```

Comment:
Why create an additional variable? Just use the constant

Reply:
On the off chance it may need to be overridden. For example, this affects gutlunches, and I don't know if it's critical for them to have a higher cap.

Reply:
Fair enough